src/reprs/runtime/utils.rkt

```diff
@@ -0,0 +1,30 @@
+#lang racket
```

Comment:
Fine though I would just put these in common (shift, unshift, Sam-append) or some plug-in common.